src/iconsManifest/supportedExtensions.ts

```diff
@@ -2238,13 +2238,8 @@ export const extensions: IFileCollection = {
     {
       icon: 'jest',
       extensions: [
-        'jest.config.json',
```

Comment:
Is `jest.config.cjs` a real option for this file? If so, could you provide some documentation about it?